containerize_leaf.sh

```diff
@@ -21,22 +21,46 @@ if [ -z ${KEYS_PATH+x} ]; then echo "Couldn't find cert+key path! Are you sure L
 #--------------
 # DB
 #--------------
-docker run -d -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=$SA_PASSWORD" -p 1433:1433 --name leaf_db_demo mcr.microsoft.com/mssql/server:2017-latest 
+docker run -d -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=$SA_PASSWORD" -p 1433:1433 \
+                -v sqlvolume:/var/opt/mssql \
+                --name leaf_db_demo mcr.microsoft.com/mssql/server:2017-latest 
+
 sleep 10s
 
 docker_sqlcmd() {
     path="$1"
     shift  # get the remaining arguments
-    docker run \
+    docker run --rm\
       -v "$PWD"/src/db/build/:/sql \
       mcr.microsoft.com/mssql-tools \
       /opt/mssql-tools/bin/sqlcmd -S 'host.docker.internal' -U SA -P "$SA_PASSWORD" "$@" -i /sql/"$path"
 }
 
+docker_getDbs(){
+  echo "Checking DBs"
+  docker run --rm \
+         -v "$PWD"/src/db/build/:/sql \
+         mcr.microsoft.com/mssql-tools \
+         /opt/mssql-tools/bin/sqlcmd -S 'host.docker.internal' -U SA -P "$SA_PASSWORD" -Q "sp_databases" > contdbs
+}
+
+docker_getDbs
+if grep -q LeafDB contdbs; then
```

Comment:
More idiomatic shell scripting: use pipes.

```suggestion
         /opt/mssql-tools/bin/sqlcmd -S 'host.docker.internal' -U SA -P "$SA_PASSWORD" -Q "sp_databases"
}

if docker_getDbs | grep -q LeafDB; then
```